heron/api/src/java/org/apache/heron/streamlet/impl/StreamletImpl.java

```diff
@@ -522,9 +523,23 @@ public void toSink(Sink<R> sink) {
   public <T> Streamlet<T> applyOperator(IStreamletOperator<R, T> operator) {
```

Comment:
need some comments to specify when `NoneStreamGrouping()` is suitable for use

Reply:
Done.